Shopify/cli
# Refactor vscode helpers from cli-kit

### WHY are these changes introduced?

Related to https://github.com/Shopify/internal-cli-foundations/issues/491

### WHAT is this pull request doing?

- Move vscode helpers to public/node

### How to test your changes?

Create an hydrogen storefront and check that `.vscode/extensions.json` contains some recommendations

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
